linux/power_operation_script/set_power_script.yml

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   Set and check soft power operation
+# # Parameters:
+#   power_ops: power script operation
+#   power_script_type: default or custom
+
+- name: "Start test {{ power_script_type }} {{ power_ops }}"
+  ansible.builtin.debug:
+    msg:
+      - "Start test {{ power_script_type }} {{ power_ops }}"
```

Comment:
Please update the task name and msg to `Start to test {{ power_script_type }} {{ power_script_op }} script`